scripts/build/TestImpactAnalysis/tiaf_persistent_storage_s3.py

```diff
@@ -48,6 +48,13 @@ def __init__(self, config: dict, suite: str, s3_bucket: str, branch: str):
             for object in self._bucket.objects.filter(Prefix=self._historic_data_key):
                 logger.info(f"Historic data found for branch '{branch}'.")
 
+                # Archive the existing object with the name of the existing last commit hash
+                # NOTE: currently disabled due to inadequate privilages for the role 'LY-Developers-CodeCommit'
+                #archive_key = f"{self._dir}/archive/{self._last_commit_hash}.{object_extension}"
+                #logger.info(f"Archiving existing historic data to '{archive_key}' in bucket '{self._bucket.name}'...")
+                #self._bucket.copy({"Bucket": self._bucket.name, "Key": self._historic_data_key}, archive_key)
+                #logger.info(f"Archiving complete.")
```

Comment:
You should remove the specified comment about not being able to access the `LY-Developers-CodeCommit` S3 bucket.

This PR and this code is public so mentioning specific S3 buckets should be removed

Reply:
fixed 👍